tests/integration/seed/seedsVerification.test.js

```diff
@@ -640,11 +642,11 @@ describe('SEEDS VERIFICATION', () => {
         });
 
         it('should pass if every company exists and is not duplicated', () => {
-          const someCompaniesDontExist = courseList
+          const everyCompanyExists = courseList
             .filter(c => c.format === BLENDED)
-            .some(c => c.companies.some(company => !company));
+            .every(c => c.companies.every(company => company));
 
-          expect(someCompaniesDontExist).toBeFalsy();
+          expect(everyCompanyExists).toBeTruthy();
 
           const someCompaniesAreDuplicated = courseList
             .filter(course => get(course, 'companies.length'))
```

Comment:
Idem pour les tests :  (j'arrive pas mettre le commentaire au bon endroit du doc)
- 'should pass if no e-learning course has companies field'
- 'should pass if no e-learning course has misc field'

Reply:
done